auv_interfaces/package.xml

```diff
@@ -14,4 +14,5 @@
   <depend>geometry_msgs</depend>
   <depend>std_msgs</depend>
   <exec_depend>message_runtime</exec_depend>
+  <test_depend>roslint</test_depend>
```

Comment:
Should we add <test_depend>rosunit</test_depend> ?

Reply:
Yes! Good catch. See https://github.com/ChrisScianna/ROS-Underwater-RnD/pull/93/commits/42ad2d2ba38bd98e1f49517263d0b29b1b0f9af7.